zerver/views/invite.py

```diff
@@ -156,6 +157,43 @@ def revoke_multiuse_invite(
     return json_success(request)
 
 
+@require_realm_admin
+@has_request_variables
+def edit_multiuse_invite(
+    request: HttpRequest,
+    user_profile: UserProfile,
+    prereg_id: int,
```

Comment:
This should be `invite_id`, since we use `MultiuseInvite` object for multi-use invites and not `PreregistrationUser` obejcts.

Reply:
I shifted the endpoint for the update from `PATCH /json/invites/:id` to `PATCH /json/invites/multiuse/:id`, which I originally intended to use as well.

Please verify if this is ok.

Reply:
Yeah, looks good for now to me.